lighthouse-core/gather/gatherers/image-usage.js

```diff
@@ -78,6 +80,8 @@ function collectImageElementInfo() {
       naturalHeight: Number.MAX_VALUE,
       isCss: true,
       isPicture: false,
+      usesObjectFit: window.getComputedStyle(element).getPropertyValue('object-fit') === 'cover'
```

Comment:
for CSS background images it doesn't actually matter since the equivalent is expressed as part of the background sizing condition, let's just return `false` here and avoid the extra cost